### ISSUE TITLE
Set wifi configuration

### ISSUE THREAD

**Issue by daitj** [abadiet/GarminServerLess#2]
Do you think it is possible to set Wifi configuration in the device? It would be nice to connect it to a Wifi without internet access to connect to local network to play music.

**Comment by abadiet** [comment]
Yes, it might be possible to do so. However, since I don't have a Wi-Fi Garmin device, I cannot test it irl. So, don't expect this feature to be available soon...
By the way, I am adding this feature to the roadmap.

**Comment by daitj** [comment]
I extracted the Garmin Express files in Wine, will check if I can decompile them in Linux. I will PR if I manage to figure that out.

**Comment by abadiet** [comment]
Thanks for the help! If you have any question or if you find anything interesting, please let me know
